src/utils/blocks.ts

```diff
@@ -15,6 +15,14 @@ export function getHeaderTag(size: TextSize) {
     }
 }
 
+export function hasBlockTag(content: string): boolean {
+    const blockElementRegex =
+        /<(div|p|h[1-6]|address|article|aside|blockquote|canvas|dd|dl|dt|fieldset|figcaption|footer|header|hr|main|nav|section|video|pre|figure|form|pre|ol|ul|li|table|thead|tbody|tfoot|tr|th|td)\b[^>]*>/gi;
```

Comment:
let's make an aray with tags like
`BLOCK_ELEMENTS = ['div', 'p',...]`
and then `BLOCK_ELEMENTS.join('|')`